GVRf/Framework/backend_oculus/src/main/jni/ovr_framebufferobject.cpp

```diff
@@ -277,8 +244,14 @@ bool FrameBufferObject::create(const ovrTextureFormat colorFormat, const int wid
                 if (depthFormat != VRAPI_TEXTURE_FORMAT_NONE) {
                     if (resolveDepth) {
                         GL( glFramebufferTexture2D(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_TEXTURE_2D, depthTexture, 0) );
+                        if (VRAPI_TEXTURE_FORMAT_DEPTH_24_STENCIL_8 == depthFormat) {
+                            GL( glFramebufferTexture2D(GL_FRAMEBUFFER, GL_STENCIL_ATTACHMENT, GL_TEXTURE_2D, depthTexture, 0) );
+                        }
```

Comment:
when attaching depth and stencil we can use "GL_DEPTH_STENCIL_ATTACHMENT" with single call. Same applies to multiview.